hack/jenkins/cron/cleanup_and_reboot_Linux.sh

```diff
@@ -36,11 +36,77 @@ logger "cleanup_and_reboot is happening!"
 # kill jenkins to avoid an incoming request
 killall java
 
+# clean minikube left overs
+echo -e "\ncleanup minikube..."
+killall minikube >/dev/null 2>&1
+USERS="$(lslogins --user-accs --noheadings --output=USER)"
+for user in $USERS; do
+    if sudo su - $user -c "minikube delete --all --purge" >/dev/null 2>&1; then
+	echo "successfully cleaned up minikube for $user user"
+    fi
+done
+
 # clean docker left overs
-docker rm -f -v $(docker ps -aq) >/dev/null 2>&1 || true
-docker volume prune -f || true
-docker volume ls || true
-docker system df || true
+echo -e "\ncleanup docker..."
+docker kill $(docker ps -aq) >/dev/null 2>&1 || true
+docker system prune --all --volumes --force || true
```

Comment:
--all is not good, because it removed all the Cached images and that will make jenkins to redownload everything all the time and eventually hit the rate limit

Reply:
ok, i've removed `--all`